packages/xrdp/PKGBUILD

```diff
@@ -8,34 +8,38 @@
 # Contributor: Tomasz Zok <tomasz.zok [at] gmail.com>
 # Contributor: techryda <techryda at silentdome dot com>
 # Contributor: Mathias R. <pu154r@overlinux.org>
+# Contributor: Alan Beale <the.mrabz@gmail.com>
 
 pkgname=xrdp
-pkgver=0.9.19
-pkgrel=2
-pkgdesc='An open source remote desktop protocol (RDP) server.'
-url='https://github.com/neutrinolabs/xrdp'
-arch=('x86_64' 'aarch64')
+pkgver=0.9.21.1
+pkgrel=1
+pkgdesc="An open source remote desktop protocol (RDP) server"
```

Comment:
> Why would you change single quote to double quotes?

I took the entire code from the original AUR since the previous one was from the original AUR. So I preferred to not touch anything. If needed, I can switch to single quote.